### PR TITLE
Parameter added to disable pagination for [download] #3522

### PR DESCRIPTION
**Pull request by PareshRadadiya**
Added parameter inside the download shortcode that will either display the pagination or not. E.g. [download pagination=yes/no]

### CODE REVIEW

**Comment by pippinsplugins** [comment]
For #3522

**Comment by spencerfinnell** [comment]
Should be true/false rather than yes/no?

**Comment by PareshRadadiya** [comment]
An **excerpt**  parameter accept "yes" or "no". The  **thumbnails** parameter requires "true" or "false" :confused:

**Comment by pippinsplugins** [comment]
Ha, oops. Yay us for being inconsistent.

I'd prefer to stick with `true/false`

**Comment by PareshRadadiya** [comment]
Okay, I've updated parameter to `true/false`

File: includes/shortcodes.php [hunk 1]
```diff
@@ -272,20 +272,29 @@ function edd_downloads_query( $atts, $content = null ) {
...
 		'order'          => $atts['order']
 	);
 
-	if ( $query['posts_per_page'] < -1 ) {
-		$query['posts_per_page'] = abs( $query['posts_per_page'] );
+	if(  'true' === $atts['pagination']  ) {
+
+		$query['posts_per_page'] = (int) $atts['number'];
+
+		if ( $query['posts_per_page'] < -1 ) {
```
Comment (cklosowski) [inline includes/shortcodes.php]: `$query['posts_per_page'] < -1` should be `$query['posts_per_page'] < 0`. By being `< -1` would allow a value of `-1`.

**Comment by cklosowski** [comment]
See my one comment, other than that, looks good to me

**Pull request merged**